metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/bolt/ParserBolt.java

```diff
@@ -117,7 +117,7 @@ public void execute(Tuple tuple) {
       int numWritten = 0;
       if(sensorParserConfig != null) {
         List<FieldValidator> fieldValidations = getConfigurations().getFieldValidations();
-        Optional<List<JSONObject>> messages = parser.parseOptional(originalMessage);
+        Optional<List<JSONObject>> messages = parser.parseOptional(originalMessage, sensorParserConfig);
```

Comment:
Line 112.  The ConfiguredParserBolt classes ensures a call to getSensorParserConfig() always returns the latest config.  Check out that class and ConfiguredBolt, they leverage Apache Curator to listen for Zookeeper changes.